### ISSUE TITLE
Error highlighting sometimes only underlines one character

### ISSUE THREAD

**Issue by brennantaylor** [bealex/SwiftLintAppCode#30]
Here are some examples

![image](https://user-images.githubusercontent.com/2840772/33341524-14e12e44-d434-11e7-9bb6-dcb4c702128b.png)

![image](https://user-images.githubusercontent.com/2840772/33341552-261def3a-d434-11e7-9227-c3434919806f.png)

![image](https://user-images.githubusercontent.com/2840772/33341570-36e86804-d434-11e7-8262-ed858bbf12d6.png)

The line and column are reported correctly in each case but I don't know why the final custom rule bleeds over into the next line

![image](https://user-images.githubusercontent.com/2840772/33341637-644991ce-d434-11e7-94e3-7726897007e2.png)

If there are characters after it the highlighting is very odd

Just wondering if this can be improved. Sometimes the underline is on a comma or paren making it hard to see or hover for the error.

**Comment by bealex** [comment]
Hi, thanks for the examples. 

Can you provide rules that are looking like that? The reason is simple, columns/rows that SwiftLint returns are for Xcode, and sometimes they change. I try to adjust them for AppCode on per-rule basis. So, these need to be readjusted.

**Comment by brennantaylor** [comment]
Sure here is the sleep example. When I run `swiftlint` on the command line it reports the correct line+column.

```yml
  sleep:
    name: "Using sleep()"
    regex: ' sleep\(.*\)'
    message: "Rather than using sleep, extract time as a dependency instead."
    severity: error
```

**Comment by bealex** [comment]
Can you please recheck this with the last plugin version, it should be fixed.
